README.md

```diff
@@ -149,7 +150,7 @@ make test-integration    # Run the integration tests
 
 ## Licence
 
-Licensed under the [Lesser General Public License (LGPL-3.0)](LICENSE).  
```

Comment:
Same whitespace issue as in https://github.com/pa11y/pa11y/pull/224#discussion_r92574725